packages/framework/client-logger/fluid-telemetry/package.json

```
@@ -69,6 +69,14 @@
 		"check:release-tags": "api-extractor run --local --config ./api-extractor-lint.json",
 		"ci:build:docs": "api-extractor run",
 		"clean": "rimraf --glob _api-extractor-temp coverage dist lib nyc \"**/*.tsbuildinfo\" \"**/*.build.log\"",
+		"eslint": "eslint src",
+		"eslint:fix": "eslint src --fix",
```

I see this version of the scripts in some of the core packages (e.g. container-loader), I think they're more up to date.
```suggestion
		"eslint": "eslint --format stylish src",
		"eslint:fix": "eslint --format stylish src --fix --fix-type problem,suggestion,layout",
```